content/azure-managed-identity.md

```diff
@@ -0,0 +1,9 @@
+You can configure BOSH to use [Azure Managed Identities](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm) to avoid hard coding specific Azure credentials.
+Azure Managed Identities are similar to AWS instance profiles.  
+You first have to create an Azure Managed Identity, and give the proper roles/permissions required by BOSH (i.e. create/delete VM, create/delete/attach disks).
```

Comment:
This could use some more details, maybe some inspiration can be drawn from the aws-iam-instance-profile docs: https://bosh.io/docs/aws-iam-instance-profiles/#only-director